doc/faq.rst

```diff
@@ -0,0 +1,229 @@
+
+   1. :ref:`sphx_glr_examples_60_search_example_parallel_n_jobs.py`
+   2. :ref:`sphx_glr_examples_60_search_example_parallel_manual_spawning_cli.py`
+
+2. Pass a `dask client <https://distributed.dask.org/en/latest/client.html>`_. If the user passes
+   a dask client, Auto-sklearn can no longer assume that it runs in sequential mode and will use
+   a ``forkserver`` to start new processes.
+
+We therefore suggest using one of the above settings by default.
+
+Auto-sklearn is extremely memory hungry in a parallel setting
+-------------------------------------------------------------
+
+When running Auto-sklearn in a parallel setting it starts new processes for evaluating machine
+learning models using the ``forkserver`` mechanism. If not all code in the main script is guarded
+by ``if __name__ == "__main__"`` it is executed for each subprocess. If such code is for example
+loading your dataset, it is executed for every evaluation of a machine learning algorithm,
+blocking your RAM.
+
+We therefore suggest moving all code inside the main block or functions.
+
+Auto-sklearn crashes with a segmentation fault
+----------------------------------------------
+
+Please make sure that you have read and followed the :ref:`installation` section! In case
+everything is set up correctly, this is most likely due to the dependency
+`pyrfr <https://github.com/automl/random_forest_run>`_ not being compiled correctly. If this is the
+case please execute:
+
+.. code:: python
+
+    import pyrfr.regression as reg
+    data = reg.default_data_container(64)
+
+If this fails, the pyrfr dependency is most likely not compiled correctly. We advice you to do the
+following:
+
+1. Check if you can use a pre-compiled version of the pyrfr to avoid compiling it yourself. We
+   provide pre-compiled versions of the pyrfr on `pypi <https://pypi.org/project/pyrfr/#files>`_.
+3. Check if the dependencies specified under :ref:`installation` are correctly installed,
+   especially that you have ``swig`` and a ``C++`` compiler.
+2. If you are not yet using Conda, consider using it; it simplifies installation of the correct
+   dependencies.
+4. Install correct build dependencies before installing the pyrfr, you can check the following
+   github issues for suggestions: `1025 <https://github.com/automl/auto-sklearn/issues/1025>`_,
+   `856 <https://github.com/automl/auto-sklearn/issues/856>`_
+
+Log files and output
+====================
+
+Where does Auto-sklearn output files by default?
+------------------------------------------------
+
+*Auto-sklearn* heavily uses the hard drive to store temporary data, models and log files which can
+be used to inspect the behavior of Auto-sklearn. Each run of Auto-sklearn requires
+its own directory. If not provided by the user, *Auto-sklearn* requests a temporary directory from
+Python, which by default is located under ``/tmp`` and starts with ``autosklearn_tmp_`` followed
+by a random string.
```

Comment:
By default, does askl remove these directories? If yes, you should mention it. If no, you should mention that this is possible